### PR TITLE
added separate status code for occupied resource locator

### LINKED ISSUE #1112: Changing the resource locator of a page to a previously used one leads to a server error
A server error occurs when changing the resource locator to one that was already used for this page and is therefore included in the history. Should display a meaningful message instead.

Exception:
```
request.CRITICAL: Uncaught PHP Exception Sulu\Component\Content\Exception\ResourceLocatorAlreadyExistsException: "" at /myproject/vendor/sulu/sulu/src/Sulu/Component/Content/Types/Rlp/Mapper/PhpcrMapper.php line 571 {"exception":"[object] (Sulu\\Component\\Content\\Exception\\ResourceLocatorAlreadyExistsException(code: 0):  at /myproject/vendor/sulu/sulu/src/Sulu/Component/Content/Types/Rlp/Mapper/PhpcrMapper.php:571)"}
```

### PR DESCRIPTION
**Pull request by danrot**
Let the rest api return status 409 instead of 500, if the resource locator is already existing. Merge with https://github.com/sulu-io/sulu-standard/pull/525

__tasks:__

- [ ] test coverage
- [ ] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | fixes #1112 
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: src/Sulu/Bundle/AdminBundle/Resources/translations/sulu/backend.en.xlf [hunk 1]
```diff
@@ -366,10 +366,6 @@
                 <source>labels.success.content-save-desc</source>
                 <target>Node sucessfully saved</target>
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/AdminBundle/Resources/translations/sulu/backend.en.xlf]: couldnt you transfer all node related translations?

File: src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.de.xlf [hunk 2]
```diff
@@ -495,6 +495,14 @@
                 <source>search-overlay.placeholder.page</source>
                 <target>Seiten durchsuchen...</target>
             </trans-unit>
+            <trans-unit id="75435af307a7ae8f965923968a0e0f94" resname="labels.error.content-save-desc">
+                <source>labels.error.content-save-desc</source>
+                <target>Seite konnte nicht gespeichert werden.</target>
+            </trans-unit>
+            <trans-unit id="75435af3a7a7a38f968923968a0e0f94" resname="labels.error.content-save-resource-locator">
+                <source>labels.error.content-save-resource-locator</source>
+                <target>Der gegebene Resource Locator ist nicht gültig.</target>
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.de.xlf]: for me it should be "Der angegebene ?Adresse? ist nicht gültig." /cc @berndhep what did you think?

**Comment by wachterjohannes** [comment]
@danrot else ACK

File: src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.de.xlf (hunk 2 above)
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.de.xlf]: Yes, that's better, will change it

File: src/Sulu/Bundle/AdminBundle/Resources/translations/sulu/backend.en.xlf (hunk 1 above)
Comment (danrot) [inline src/Sulu/Bundle/AdminBundle/Resources/translations/sulu/backend.en.xlf]: There are even more? Will have a look again.

**Comment by danrot** [comment]
@wachterjohannes fixed

**Pull request merged**